h2o-r/tests/testdir_jira/runit_pubdev_4739.R

```diff
@@ -0,0 +1,17 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+
+test.pubdev_4739 <- function() {
+    voting.url <- "http://archive.ics.uci.edu/ml/machine-learning-databases/voting-records/house-votes-84.data"
```

Comment:
Please try to use `/smalldata` directory for fetching datasets. URL's are not the most stable solution.